src/index.html

```diff
@@ -5,9 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
```

Comment:
Kindly change title to reflect the project.

Reply:
- ok thank you i will do it now